package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "rsvp-spec",
+  "version": "1.0.0",
+  "description": "RSVP spec.",
+  "main": "js",
+  "module": "./dist/module.js",
+  "files": [
+    "schemas/",
+    "dist/module.js",
+    "js/schemas.js",
+    "js/index.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:digitalbazaar/rsvp-spec.git"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/rsvp-spec/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/rsvp-spec/#readme",
+  "license": "SEE LICENSE IN LICENSE.md",
+  "devDependencies": {
+    "eslint": "^7.0.0",
+    "eslint-config-digitalbazaar": "^2.4.0",
+    "json-loader": "^0.5.7",
+    "rollup": "^2.10.5",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.11"
+  },
+  "scripts": {
+    "prepublish": "npm run build",
+    "build": "webpack && rollup -c",
+    "lint": "eslint *.js js"
+  },
+  "publishConfig": {
+    "registry": "https://npm.pkg.github.com/"
```

Comment:
Is this just copy & paste?  Seems like it should go to default npm registry.

Reply:
This is what we are using in other repos. Should we not being using this?

Reply:
We only need the publishconfig for the github registry on private packages.